dogs-initial/src/main/java/com/lambdaschool/dogsinitial/DogsinitialApplication.java

```diff
@@ -1,17 +1,70 @@
 package com.lambdaschool.dogsinitial;
 
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.DispatcherServlet;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
+@EnableWebMvc
 @SpringBootApplication
 public class DogsinitialApplication
 {
+    public static final String EXCHANGE_NAME = "LambdaServer";
+    public static final String QUEUE_NAME_LOW = "LowPriorityQueue";
+    public static final String QUEUE_NAME_HIGH = "HighPriorityQueue";
+
```

Comment:
Instead of naming the queues as "high"/"low" priority, you could give clearer names that indicate the endpoint information they are supposed to contain.